packages/cli/src/lib/index.js

```diff
@@ -326,7 +326,7 @@ export const addRoutesToRouterTask = (routes) => {
     if (content.includes(route)) {
       return content
     }
-    return content.replace(/(\s*)\<Router\>/, `$1<Router>$1  ${route}`)
+    return content.replace(/<Router>(\s*)/, `<Router>$1${route}$1`)
```

Comment:
Reading my own PR I finally understand how this is supposed to work. Will have to revert this change and update my test instead...

Reply:
Keeping this as it's a bit shorter, and a bit more flexible as well (with the indentation)